pkg/apis/servicecatalog/validation/bindingvalidation.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2017 The Kubernetes Authors.
```

Comment:
sounds good, agree, will change. main reason to split was to avoid any git conflicts if people want to split the work up.